pkg/kubectl/cmd/create_service.go

```diff
@@ -83,6 +82,10 @@ func NewCmdCreateServiceClusterIP(f cmdutil.Factory, cmdOut io.Writer) *cobra.Co
 	return cmd
 }
 
+func errUnsupportedGenerator(cmd *cobra.Command, generatorName string) error {
```

Comment:
It's only used by the functions in this package, so I kept it as an unexported function. I'm trying to avoid cross-package dependencies where it's not absolutely necessary :)
If you feel strongly on the issue I will move it, no problem.